lib/src/rules/avoid_returning_null_for_void.dart

```diff
@@ -89,14 +89,14 @@ class _Visitor extends SimpleAstVisitor<void> {
     bool? isAsync;
     if (parent is FunctionExpression) {
       type = parent.declaredElement?.returnType;
-      isAsync = parent.body?.isAsynchronous;
+      isAsync = parent.body.isAsynchronous;
```

Comment:
Looks like a dup of the above.
Consider adding a helper to get the `ExecutableElement` and use in both places to access the return type and `isAsynchronous`.